rust/content.md

```diff
@@ -39,8 +39,8 @@ COPY . .
 RUN cargo install --path .
 
 FROM debian:buster-slim
-RUN apt-get update && apt-get install -y libssl
-COPY --from=builder /usr/src/myapp /usr/local/bin/myapp
+RUN apt-get update && apt-get install -y libssl-dev
```

Comment:
I don't think the development package is required in this image.

Reply:
What is the non-dev version of libssl-dev? There is no libssl.

Reply:
I think the line is there to show how one could install required packages. A simple myapp probably would not require openssl.

Reply:
So maybe both should be replaced by a dummy package name that doesn't exist to further illustrate that it's merely an example?

Reply:
Sorry - `libssl1.1`. Yeah, the line is intended to show how you'd pull in extra runtime dependencies, OpenSSL just being one relatively common example.

Reply:
Sure, we could swap it out to `extra-runtime-dependencies` or something like that.

Reply:
Ok. Makes sense.